implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -206,6 +207,14 @@ private static JWTAuthContextInfoProvider create(String publicKey, String public
     @ConfigProperty(name = "smallrye.jwt.expiration.grace", defaultValue = "60")
     private Optional<Integer> expGracePeriodSecs;
 
+    /**
+     * The maximum number of seconds that a JWT may be issued for use. Effectively, the difference
+     * between the expiration date of the JWT and the issued at date must not exceed this value.
+     */
+    @Inject
+    @ConfigProperty(name = "smallrye.jwt.time-to-live.max")
+    Optional<Integer> maxTimeToLiveSecs;
```

Comment:
Lets have it as `Optional<Long>` just in case (see Jose4J `NumericDate`)

Reply:
Updates pushed